sdk/keyvault/keyvault/test/client_key_POC.c

```diff
@@ -13,6 +16,39 @@
 
 int exit_code = 0;
 
+az_span get_key_version(az_http_response * response) {
+  az_http_response_parser parser = { 0 };
+  az_span body = { 0 };
+  az_result r = az_http_response_parser_init(&parser, az_span_builder_result(&response->builder));
+
+  az_http_response_status_line status_line = { 0 };
+  r = az_http_response_parser_read_status_line(&parser, &status_line);
+
+  // Get Body
+  r = az_http_response_parser_skip_headers(&parser);
+  r = az_http_response_parser_read_body(&parser, &body);
+
+  // get key from body
+  az_json_value value;
+  r = az_json_get_by_pointer(body, AZ_STR("/key/kid"), &value);
+
+  az_span k = { 0 };
+  r = az_json_value_get_string(&value, &k);
+
+  // calculate version
```

Comment:
I don't know what the raw JSON looks like for this but isn't there a json parsing function (created by @sergey-shandar) that can just be called here?

Reply:
yes, that is `az_json_get_by_pointer` and it's used before this.
We have an `az_http_response` that contains an az_span_builder that eventually writes headers and body of a http response.
Then, we have http_response_parser to get the body
Then, body is Json, so we use Json parser to get one value...

And here, from that value (string), I need the last part of it